dask_kubernetes/helm.py

```diff
@@ -41,8 +41,11 @@ class HelmCluster(Cluster):
         Name of the Dask worker deployment in the current release.
         Defaults to "worker".
     **kwargs: dict
-        Additional keyword arguments to pass to Cluster
-
+        nodeport_host string (optional)
+            A Node address. Can be provided in case scheduler service type is NodePort
+            and you want to avoid listing Nodes at a cluster scope.
+        node_port: int (optional)
+            A NodePort for your scheduler service.
```

Comment:
I think the ordering and indentation is off here.

Reply:
Adjusted docstrings a little bit to be more compact. Not sure what should be an appropriate way of documenting `kwargs`. Please advise me if it looks ok now.